web/app/components/new/template-form.hbs

```diff
@@ -0,0 +1,104 @@
+{{! @glint-nocheck: not typesafe yet }}
```

Comment:
I have copied from doc-form.hbs to maintain the same UI experience and then added my required fields
1. Template ID 
2. Template long name
3. Description
4. Document link